src/main.ts

```diff
@@ -318,6 +318,11 @@ const command: Command = {
 				);
 			}
 		});
+
+		options('staticOnly', {
```

Comment:
I don't think we should add this to the options or dojorc?

Reply:
Good point